src/common/cockpitwebcertificate.c

```diff
@@ -148,113 +140,3 @@ cockpit_certificate_key_path (const char *certfile)
 
   return keypath;
 }
-
-/**
- * cockpit_certificate_parse:
- *
- * Load the ws certificate file, and split it into the private key and
- * certificates PEM strings.
- *
- * Return 0 on success, or -errno on failure. This can be an error from open()
- * or read(), or ENOKEY if private key is missing.
- */
-int
-cockpit_certificate_parse (const char *file, char **cert, char **key)
```

Comment:
As you keep resolving my previous comment, adding a new one: The declaration from src/common/cockpitwebcertificate.h also must be dropped.

Reply:
I didn't see this comment yet.  Thanks :)